test/functional/test_framework/mempool_util.py

```diff
@@ -30,6 +30,11 @@
     MiniWallet,
 )
 
+ORPHAN_TX_EXPIRE_TIME = 1200
```

Comment:
Orphans don't expire anymore and this is unused (is it from a rebase maybe?)

Reply:
I've removed it, pretty sure it was from a rebase